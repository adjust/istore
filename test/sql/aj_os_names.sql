SELECT 'android'::os_name;
SELECT 'ios'::os_name;
SELECT 'windows'::os_name;
SELECT 'windows-phone'::os_name;
SELECT 'unknown'::os_name;
SELECT 'kermit'::os_name;
