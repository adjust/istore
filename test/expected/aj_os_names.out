SELECT 'android'::os_name;
 os_name 
---------
 android
(1 row)

SELECT 'ios'::os_name;
 os_name 
---------
 ios
(1 row)

SELECT 'windows'::os_name;
 os_name 
---------
 windows
(1 row)

SELECT 'windows-phone'::os_name;
    os_name    
---------------
 windows-phone
(1 row)

SELECT 'unknown'::os_name;
 os_name 
---------
 unknown
(1 row)

SELECT 'kermit'::os_name;
 os_name 
---------
 unknown
(1 row)

